### PR TITLE
Improve learn section

### PR DESCRIPTION
**Pull request by silvashenol20**
This PR is to update the Accounts page according to the feedback from @imesh94 

- Fix configuration issues

### CODE REVIEW

File: en/docs/try-out/consent-manager.md [hunk 1]
```diff
@@ -56,7 +45,7 @@ Follow [Configuring users and roles](../install-and-setup/configuring-users-and-
 6. Set the value for **Endpoint** as follows:
 
     ``` 
-    https://<IS_HOST>:9446/api/openbanking/consent
+    https://localhost:9443/api/openbanking/cds/backend/services
```
Comment (isharailanga) [inline en/docs/try-out/consent-manager.md]: no need to change this URL (unless specified for consent mgr portal)

File: en/docs/try-out/consent-manager.md [hunk 2]
```diff
@@ -18,17 +18,6 @@ requirements and manages consents.
     http_method="GET,DELETE"
     ```
    
-2. Open the `<API_HOME>/repository/conf/deployment.toml` file and add the following gateway executor configurations for
```
Comment (isharailanga) [inline en/docs/try-out/consent-manager.md:21]: unless specified in the developer docs, let's not remove these configs as well

File: en/docs/try-out/consent-manager.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/try-out/consent-manager.md]: fixed in https://github.com/wso2/docs-ob-toolkit-cds/pull/19/commits/eeb820c41b402329079378586375b7a780a4a8aa

File: en/docs/try-out/consent-manager.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/try-out/consent-manager.md:21]: fixed in https://github.com/wso2/docs-ob-toolkit-cds/pull/19/commits/eeb820c41b402329079378586375b7a780a4a8aa

**Comment by silvashenol20** [comment]
Fixed the from this [https://github.com/wso2/docs-ob-toolkit-cds/pull/20](https://github.com/wso2/docs-ob-toolkit-cds/pull/20) PR

**Pull request closed without merging**